### PR TITLE
Using Bootstrap: Newsweek.com

### PR DESCRIPTION
**Pull request by eananti**
In this project, we created a replica of Newsweek.com official homepage using HTML, CSS and Bootstrap.

Additional Information:

- Setup project dependencies like GitHub Actions and Linters via Microverse Linters Config.
- Defined a well structured DOM with each element properly grouped together using proper sizes and alignments.
- Used the browser's console to review the page's development strategy.
- Used semantic tags and others were necessary.
- Deployed a mobile-first approach in building a responsive website with various breakpoints.
- Used Bootstrap in defining the layout and styling most elements.

### CODE REVIEW

File: css/media/max-width.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@media all and (max-width: 768px) {
+  .subscribe-btn {
+    font-size: 0.8em;
+    padding: 0.5em;
+    margin-right: 0 !important;
```
Comment (umairarshadbutt) [inline css/media/max-width.css]: Remove` !important`

File: css/media/max-width.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    margin-left: 0.4em !important;
```
Comment (umairarshadbutt) [inline css/media/max-width.css]: Remove `!important`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @eananti AND @GzimAsani 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] No !important overusing in css. [Article for support](https://speckyboy.com/good-bad-css-practices/)

- [x] Please fix the order for the small screen approach
This is the correct order

- Top story
- Featured stories
- The debate
- Opinion
- More Stories
- My Turn
- Culture& Travel
- In the magazine
- Editor's pick
- Footer



### Subscribe
|[Source Page](https://www.newsweek.com/)|[Clone Page](https://eananti.github.io/News-Week-Clone/)|
|:---:|:---:|
|original-sub|clone-sub|

- [x] Manage content on a medium screen.


### Footer
|[Source Page](https://www.newsweek.com/)|[Clone Page](https://eananti.github.io/News-Week-Clone/)|
|:---:|:---:|
|original-footer|clone-footer|

- [x] Align `© 2020 Newsweek` as original website on a medium screen.

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by ghost — CHANGES_REQUESTED** [review]
**Hi @eananti AND @GzimAsani**,

Good job on your project. However, there are still some changes required.

### **Changes Required:**
1) **There is an image that is not displaying properly, as shown below in the blue circled area.**

2020-09-16 19_38_52-image_broken


2) **Your navbar, at 992px width, doesn't match the original.**

**clone**     |   **original**
--------  | ------------
navbar_clone  |  navbar_original

### **Optional changes:**
To help make your webpage look even better:

1) Fix the size of the images so they don't look too vertically stretched. Example:

-  The images at my turn and cultural&travel sections, at 768px width.

-  The images at in the magazine section, at 992px width.

2). You can reduce the font-size of the headings or add more width in the cultural & travel section, for the small screen sizes, so the wordings don't look too long.

### **Good luck!**

**Review by ghost — APPROVED** [review]
### **Congratulations. Your project has been approved!** 👏.
 
### **Good luck on your next project.**

**Pull request merged**